app-misc/opentrack/opentrack-9999.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop flag-o-matic toolchain-funcs
+
+DESCRIPTION="Head tracking software for MS Windows, Linux, and Apple OSX"
+HOMEPAGE="https://github.com/opentrack/opentrack"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/opentrack/opentrack.git"
+else
+	SRC_URI="https://github.com/opentrack/opentrack/archive/refs/tags/${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/opentrack-${P}"
+fi
+
+LICENSE="ISC"
+SLOT="0"
+
+IUSE="neuralnet wine opencv openmp"
+
+REQUIRED_USE="neuralnet? ( openmp opencv )"
+
+DEPEND="
+	neuralnet? ( sci-libs/onnxruntime-bin )
+	opencv? ( media-libs/opencv:= )
+	wine? ( virtual/wine )
+	dev-libs/libevdev
+	dev-libs/libusb:1
+	dev-qt/qtbase:6
```

Comment:
```suggestion
	dev-qt/qtbase:6[gui,network,widgets]
```

Reply:
Thanks, I didn't know Qt6 moved this to use flags. Fixed.